experimental/packages/exporter-trace-otlp-http/package.json

```
@@ -72,6 +72,7 @@
     "babel-loader": "8.2.3",
     "codecov": "3.8.3",
     "cpx": "1.5.0",
+    "fetch-mock": "^9.11.0",
```

Let's pin this instead, we've run into some troubles with unpinned dependencies before :slightly_smiling_face: 
```suggestion
    "fetch-mock": "9.11.0",
```